03-Node-npm-modules/js/Movie.js

```diff
@@ -0,0 +1,30 @@
+define(['Director'], function(Director) {
+
+	var Movie = function (title) {
+		this.title = title;
+
+	};
+	Movie.prototype = {
+
+		play:function(){ return "Playing: " + this.title + "..."; },
+		stop:function(){ return "Stoping: " + this.title + "..."; },
+		set:function(attr, value){ this[attr] = value; },
+		get:function(attr){
+			return this[attr]; }
+		};
+
+		function extend(destination, source) {
+			for (var k in source) {
+				if (source.hasOwnProperty(k)) {
+					destination[k] = source[k];
+				}
+			}
+			return destination; 
+		}
+
+		extend(Movie.prototype, Director);
```

Comment:
You don't have to extend Movie with Director methods. Moreover, I'm not sure that you will extend anything, because Director doesn't have any own method, just methods shared through its prototype, but it's worth being checked. Anyway, you can remove this line I think :)